src/ui/FileViewer/ToggleHeader/ToggleHeader.jsx

```diff
@@ -1,23 +1,27 @@
 import PropTypes from 'prop-types'
 
+import ComponentsMultiSelect from 'pages/RepoPage/CoverageTab/subroute/ComponentsMultiSelect'
 import { LINE_STATE } from 'shared/utils/fileviewer'
 
 import Title, { TitleCoverage, TitleFlags, TitleHitCount } from './Title/Title'
-
 function ToggleHeader({
   title,
   sticky = false,
   showHitCount = false,
   showFlagsSelect = false,
+  showComponentsSelect = false,
 }) {
   return (
-    <Title title={title} sticky={sticky}>
-      <TitleCoverage coverage={LINE_STATE.UNCOVERED} />
-      <TitleCoverage coverage={LINE_STATE.PARTIAL} />
-      <TitleCoverage coverage={LINE_STATE.COVERED} />
-      <TitleHitCount showHitCount={showHitCount} />
-      {showFlagsSelect && <TitleFlags />}
-    </Title>
+    <div>
```

Comment:
Also just for my knowledge, why does this need to be `div` wrapped?

Reply:
So wrapping it in a `div` means that the styles from the parent are only applied to the outer element, whereas the inner elements are unchanged by the outer styles. If you want a big longer of an explanation, let me know!

Reply:
@nicholas-codecov oh absolutely! I've always just shoved things into a div to make it work, but I never knew the real reasoning